fatalerror.cpp

```diff
@@ -69,7 +69,7 @@ LPSTR __stdcall GetErrorText(DWORD win32ErrorCode)
 void __stdcall vtrace(const char* szFormat, va_list args)
 {
 	char buf[1024];
-	::vsprintf(buf, szFormat, args);
+	::vsprintf_s(buf, sizeof(buf), szFormat, args);
```

Comment:
May as well leave this one as you've done it since the format arg is const char*.